src/generated/resources/assets/surviving_the_aftermath/lang/en_us.json

```diff
@@ -1,5 +1,7 @@
 {
   "block.surviving_the_aftermath.example": "Example",
   "itemGroup.surviving_the_aftermath": "Surviving the Aftermath",
-  "surviving_the_aftermath.nether_raid": ""
+  "surviving_the_aftermath.nether_raid": "",
```

Comment:
The language json files should not be edited manually, make changes to ModLanguageCNProvider.java and ModLanguageProvider.java instead